packages/styles/scss/components/leaving-ibm/_leaving-ibm.scss

```
@@ -0,0 +1,56 @@
+//
+// Copyright IBM Corp. 2020, 2023
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '@carbon/styles/scss/config' as *;
+@use '@carbon/styles/scss/spacing' as *;
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/type' as *;
+@use '../../../../carbon-web-components/src/components/modal/modal' as *;
+@use '@carbon/ibmdotcom-styles/scss/globals/vars' as *;
+
+@mixin leaving-ibm {
+  :host(#{$c4d-prefix}-leaving-ibm-composite),
+  :host(#{$c4d-prefix}-leaving-ibm-container) {
+    width: 100%;
+  }
+
+  :host(#{$c4d-prefix}-leaving-ibm-modal) {
+    @extend :host(#{$prefix}-modal);
+
+    max-height: 100%;
+
+    &[open] {
+      @extend :host(#{$prefix}-modal[open]);
+    }
+  }
```

```suggestion
  :host(#{$c4d-prefix}-leaving-ibm-modal) {
    @extend :host(#{$prefix}-modal);

    max-height: 100%;
  }

  :host(#{$c4d-prefix}-leaving-ibm-modal[open]) {
      @extend :host(#{$prefix}-modal[open]);
  }  
```

Due to a previous issue we've encountered, doing `&[open]` might not work in some environments, so we're having these properties be its own declarations within `:host()` .